app/Listeners/Hold/UnassignHoldOnDisconnect.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+
+namespace App\Listeners\Hold;
+
+use App\Events\HoldUnassignedEvent;
+use App\Models\Vatsim\NetworkAircraft;
+
+class UnassignHoldOnDisconnect
+{
+    public function handle(NetworkAircraft $aircraft) : bool
+    {
+        event(new HoldUnassignedEvent($aircraft));
+        return false;
```

Comment:
Does the false help indicate something which could be done via exception flow?

Reply:
No - the false is required by Laravel to tell it not to propagate the event any further :)